source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -129,6 +254,114 @@ This means that in order to have a VM connected to both ipv4 and ipv6 subnets,
 it needs to connect to 2 different networks (through 2 different logical ports),
 one with an IPv4, other with an IPv6 subnet defined on top.
 
+### Networking API entity provisioning
+
+Below you can find the steps required to have dynamic IPv6 addresses on the
+guest VMs, along with the available methods of provisioning it.
+
+1. create a logical network
+    - oVirt UI
+    - oVirt REST API
+    - manageIQ UI (*)
+    - ovirt-provider-ovn REST API. (*) [REST example](#create-a-logical-network)
+2. add a subnet on top
+    - oVirt UI - *requires changes*
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-subnet)
+3. provision a router
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-router)
+4. attach the subnet to the router
+    - manageIQ UI
+    - oVirt ovirt-provider-ovn REST API. [REST example](#configure-the-router)
+5. create a VM having the vNIC profile of the network created in step #1
+   **or** assign the vNIC profile of the network created in #1 to an existing
+   VM
+    - oVirt UI
+    - oVirt REST API
+
+(*) - when the logical network is created via provider's REST API, of through
+manageIQ, it is necessary to import the logical network into oVirt engine.
+
+#### EL7 Guest VM configuration
+
+For dhcpv6-stateless the guest VM can be configured using *either* of the 3
+configuration options defined in the [Guest config section](#guest-vm-configuration).
+
+It is important to state that if and when the dhcp client version supplied in
+EL7 stops defaulting to /64 prefixes, the guest VMs have to be configured as
+indicated for [EL8](#el8-guest-vm-configuration).
+
+#### EL8 Guest VM configuration
+
+On the version of dhcp client supplied in el8, dhcpv6 alone is **not enough** to
+configure dynamic IP addresses on the guest VMs.
+
+This means that for stateless dhcpv6, the guest VM is configured using *either*
+option [Linux Kernel RA configuration](#linux-kernel-ra-configuration) **or**
+option [NetworkManager](#networkmanager), whereas for stateful dhcpv6, the
+configuration **has to** follow options [DHCPv6](#dhcpv6) **and** [Linux Kernel RA configuration](#linux-kernel-ra-configuration).
+
```

Comment:
> whereas for stateful dhcpv6, the
configuration **has to** follow options [DHCPv6](#dhcpv6) **and** [Linux Kernel RA configuration](#linux-kernel-ra-configuration).

I guess this is not true for network manager usage.
From my point of view, you are trying to achive very much here. If you would just write that RHEL8 works out of the box, I think this would be enough.

Reply:
OK, I'll simplify the wording.

Reply:
Done